js/flightlog_fields_presenter.js

```diff
@@ -459,7 +459,7 @@ function FlightLogFieldPresenter() {
         return eventState;
     }
     
-    function presentEnum(value, enumNames) {
+    FlightLogFieldPresenter.presentEnum = function presentEnum(value, enumNames) {
```

Comment:
I know that having this as an instance member is a pattern that's already used in here, so doing it this way is fine.

But in general, defining functions that are innate parts of a class as prototype members is a bit cleaner.